vercel.json

```diff
@@ -14,6 +14,10 @@
 		{
 			"source": "/ngx-admin/(.*)",
 			"destination": "/ngx-admin/index.html"
+		},
+		{
+			"source": "/turbolinks/(.*)",
+			"destination": "/turbolinks/index.html"
```

Comment:
As mentioned in my comment, this does not apply here, it's not doing anything.

Reply:
- [ ] Remove unnecessary configuration

Reply:
Let me review